presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/glue/TestGlueHiveMetastoreConfig.java

```diff
@@ -54,6 +56,8 @@ public void testExplicitPropertyMapping()
                 .put("hive.metastore.glue.aws-credentials-provider", "custom")
                 .put("hive.metastore.glue.catalogid", "0123456789")
                 .put("hive.metastore.glue.use-instance-credentials", "true")
+                .put("hive.metastore.glue.total-segments", "10")
```

Comment:
These need to be updated to match the new names. This is the cause of the Travis CI failure.

Reply:
Fixed. Sorry for missing that.